plugins/node/instrumentation-undici/README.md

```diff
@@ -25,7 +25,9 @@ OpenTelemetry Undici/fetch Instrumentation allows the user to automatically coll
 To load a specific instrumentation (Undici in this case), specify it in the Node Tracer's configuration.
 
 ```js
-const { UndiciInstrumentation } = require('@opentelemetry/instrumentation-undici');
+const {
+  UndiciInstrumentation,
```

Comment:
my prettier formatter did these changes, lmk if you want them to be reverted.

Reply:
All good.